### PR TITLE
Confirm keyboard

### PR DESCRIPTION
**Pull request by nkochnev**

### CODE REVIEW

**Comment by dohly** [comment]
Please change target branch to develop

File: DemoBot/DemoBot.fsproj [hunk 1]
```diff
@@ -7,6 +7,7 @@
 
   <ItemGroup>
     <Compile Include="Examples\EmbraerE170Reservations.fs" />
+    <Compile Include="Examples\ConfirmKeyborad.fs" />
```
Comment (dohly) [inline DemoBot/DemoBot.fsproj]: typo in filename

File: DemoBot/Examples/ConfirmKeyborad.fs [hunk 2]
```diff
@@ -0,0 +1,29 @@
+namespace Funogram.Keyboard
+module ConfirmKeyborad=
+    open System
+    open System.Globalization   
+    open Funogram.Keyboard.Inline
+    [<Literal>]
+    let private CONFIRM="CONFIRM"
+    let private keyboard (keys:KeyboardBuilder<bool>) (b:bool)= 
+               let X=keys.Ignore
```
Comment (dohly) [inline DemoBot/Examples/ConfirmKeyborad.fs]: you don't need this one

**Review by dohly — CHANGES_REQUESTED** [review]

File: DemoBot/DemoBot.fsproj (hunk 1 above)
Comment (nkochnev) [reply DemoBot/DemoBot.fsproj]: fixed

File: DemoBot/Examples/ConfirmKeyborad.fs (hunk 2 above)
Comment (nkochnev) [reply DemoBot/Examples/ConfirmKeyborad.fs]: deleted CONFIRM
what about `let private keyboard (keys:KeyboardBuilder<bool>) (b:bool)= 
               let X=keys.Ignore` ?

Comment (dohly) [reply DemoBot/Examples/ConfirmKeyborad.fs]: I meant you don't need X and B

File: DemoBot/DemoBot.fsproj (hunk 1 above)
Comment (nkochnev) [reply DemoBot/DemoBot.fsproj]: fixed

Comment (dohly) [reply DemoBot/DemoBot.fsproj]: did you pushed the changes?

File: DemoBot/Examples/ConfirmKeyborad.fs (hunk 2 above)
Comment (nkochnev) [reply DemoBot/Examples/ConfirmKeyborad.fs]: fixed

**Review by dohly — APPROVED** [review]

File: DemoBot/Examples/ConfirmKeyborad.fs (hunk 2 above)
Comment (nkochnev) [reply DemoBot/Examples/ConfirmKeyborad.fs]: I moved `keyborad` function to `GetKeysByState` like `EmbraerE170Reservations`

**Pull request merged**